### PR TITLE
[ADD] mozaik_membership_sepa_payment_return

### PR DESCRIPTION
**Pull request by marielejeune**
refs #66042

### CODE REVIEW

File: mozaik_membership_sepa_payment_return/models/payment_return.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                    body=_(
+                        "Trying to recognize a partner. Result: "
+                        "No bank account found for this account number "
+                        "and partner combination."
+                    )
+                )
+            else:
+                payment_return.message_post(
+                    body=_(
+                        "Trying to recognize a partner. Result: " "Partner recognized."
```
Comment (qgroulard) [inline mozaik_membership_sepa_payment_return/models/payment_return.py]:
```suggestion
                        "Trying to recognize a partner. Result: Partner recognized."
```

File: mozaik_membership_sepa_payment_return/models/payment_return.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                )
+                continue
+            if not membership_line.paid:
+                payment_return._set_to_error(
+                    _(
+                        "Active membership line is not paid. Please process this line manually."
+                    )
+                )
+                continue
+            if not abs(payment_return.amount) == abs(membership_line.price):
```
Comment (qgroulard) [inline mozaik_membership_sepa_payment_return/models/payment_return.py]: From my understanding `payment_return.amount` should not be transformed to absolute value.

File: mozaik_membership_sepa_payment_return/models/payment_return.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+        self._send_notification_email()
+        self.write({"state": "done", "error_message": False})
+
+    def _process_refusal(self):
+        """
+        Filter payment returns and call _process_filtered_refusal on the
+        ones that can be processed automatically.
+        """
+        self._filter_payment_returns()._process_filtered_refusal()
```
Comment (qgroulard) [inline mozaik_membership_sepa_payment_return/models/payment_return.py]: I don't really see the point of this method.

**Review by qgroulard — COMMENTED** [review]
Great work !
Minor comments

File: mozaik_membership_sepa_payment_return/models/payment_return.py (hunk 3 above)
Comment (marielejeune) [reply mozaik_membership_sepa_payment_return/models/payment_return.py]: As discussed together, we will extend _process_filtered_refusal (now called _process_refusal) in Groen to deal with NationBuilder tags.

**Pull request merged**